src/NavigationExperimental/NavigationAnimatedValueSubscription.js

```diff
@@ -22,24 +22,26 @@
  * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THE SOFTWARE, EVEN IF
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
- * @typechecks
```

Comment:
Lol, that git diff is weird. Actually deleted clamp.js and used a version from npm.